build-all-targets.sh

```diff
@@ -18,6 +18,15 @@ SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
 
 CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
 
+# Check if a certai feature set was requested
+if [ -z "$KUKSA_DATABROKER_FEATURES" ]; then
```

Comment:
We should better document (in our wiki? As part of release testing? ) how to set this variable when building locally to get viss support. The only use-case I know of right now for this shell script is if I want to build docker containers locally, as our Dockerfile relies on that binaries already exist.

Might spend some time trying this out unless you have already tested thoroughly Sebastian

Reply:
It has been used by me :-) 

In my view it is beneficial to have one way to build docker containers locally, good if doing major changes